fedimint-api/src/module/mod.rs

```diff
@@ -278,6 +280,10 @@ pub trait IModuleGen: Debug {
         task_group: &mut TaskGroup,
     ) -> anyhow::Result<DynServerModule>;
 
+    /// Retreives the `DatabaseMigrationData` from the module to be applied to the database
+    /// before the module is initialized.
+    fn get_database_migration_data(&self) -> DatabaseMigrationData;
```

Comment:
"data" is a word to avoid. Everything in SWE is data, so it doesn't bring any information.

Reply:
Removed :)